src/main/java/io/confluent/idesidecar/restapi/resources/ConnectionsResource.java

```diff
@@ -60,10 +60,12 @@ public Uni<ConnectionsList> listConnections() {
                   .completionStage(() -> getConnectionModel(connection.getSpec().id())))
               .collect(Collectors.toList());
           if (connectionFutures.isEmpty()) {
+            Log.error("Returning no connections");
             return Uni
                 .createFrom()
-                .item(ConnectionsList.from(List.of()));
+                .item(new ConnectionsList());
           }
+          Log.errorf("Returning %d connections", connectionFutures.size());
```

Comment:
Same as above; what about changing the log level to debug?

Reply:
Same as above. It's not needed. I'll double check other log message changes.